Mathlib/Probability/CondCount.lean

```diff
@@ -21,15 +21,15 @@ for that purpose.
 
```

Comment:
Can you rename the file to `UniformOn`?

Reply:
I was afraid it would mess the diff, but looks like it went fine

Reply:
Nevermind. It did mess up the diff. Are you sure you want me to do this?